contracts/protocol/Sector3DAO.sol

```diff
@@ -39,11 +39,13 @@ contract Sector3DAO {
    */
   Sector3DAOPriority[] public priorities;
 
+  event PriorityDeployed(Sector3DAOPriority priority);
```

Comment:
@developerfred Added this as a separate PR:  https://github.com/sector-3/protocol/pull/66

Reply:
thanks @aahna-ashina 🚀